openfe/utils/handle_trajectories.py

```diff
@@ -78,7 +78,7 @@ def _create_new_dataset(filename: Path, n_atoms: int,
         AMBER Conventions compliant NetCDF dataset to store information
         contained in MultiState reporter generated NetCDF file.
     """
-    ncfile = nc.Dataset(filename, 'w', format='NETCDF3_64BIT')
+    ncfile = nc.Dataset(filename, 'w', format='NETCDF3_64BIT_OFFSET')
```

Comment:
This should fix the mypy error, see here: https://github.com/Unidata/netcdf4-python/blob/9426a684d3bd7b6c0c9b4d3c3f8531f30583d1e2/README.md?plain=1#L202C37-L202C57